.github/pull_request_template.md

```diff
@@ -31,20 +31,21 @@ Pick one of the following and delete the others:
 <!--
 Please select the tests that you ran to verify your changes:
```

Comment:
Change it to "Pick one of the following and delete the others:" as well?

Reply:
I think this is multiple choice not a single choice.